eo-maven-plugin/src/test/java/org/eolang/maven/optimization/OptCachedTest.java

```diff
@@ -50,73 +49,81 @@ final class OptCachedTest {
     /**
      * Test case for XML program in cache.
      *
-     * @param tmp Temp dir
+     * @param cache Temp cache dir.
+     * @param dir Temp program dir
      * @throws IOException if I/O fails
      * @todo #2422:60min returnsFromCacheIfXmlAlreadyInCache: this test is unstable.
      *  We should resolve issues with unstable failures and only
      *  then enable the test.
      *  Also, see this <a href="https://github.com/objectionary/eo/issues/2727">issue</a>.
      */
-    @Disabled
     @Test
-    void returnsFromCacheIfXmlAlreadyInCache(@TempDir final Path tmp) throws IOException {
-        final XML program = OptCachedTest.program(ZonedDateTime.now());
-        OptCachedTest.save(tmp, program);
+    void returnsFromCacheIfXmlAlreadyInCache(@TempDir final Path cache, @TempDir final Path dir)
+        throws Exception {
+        final XML xml = OptCachedTest.program();
+        final Path program = OptCachedTest.save(dir, xml, 0);
```

Comment:
@Yanich96 [Optional] You are using `OptCachedTest.save(dir, xml, 0)`  in many places. Maybe we can overload this method and allow to call just `OptCachedTest.save(dir, xml)` and pass `0` by default? What do you think?